influxdb3_write/src/catalog.rs

```diff
@@ -25,7 +25,7 @@ pub enum Error {
 
     #[error(
         "Update to schema would exceed number of tables limit of {} tables",
-        Catalog::NUM_TABLES_LIMIT
+        Catalog::NUM_TABLES_LIMIT - 1
```

Comment:
You increased the limit for NUM_COLUMNS_PER_TABLE_LIMIT not the NUM_TABLES_LIMIT constant and so this change to subtract 1 would display the wrong number

Reply:
Oops, good catch, I will fix that.